src/main/java/Elements/AbstractElement.java

```diff
@@ -0,0 +1,72 @@
+package Elements;
+
+import Utils.Wait;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public abstract class AbstractElement {
+
+    protected final Wait wait;
+    private WebDriver driver;
+    private final By locator;
+
+    public AbstractElement(final WebDriver driver, final By locator) {
+        this.driver = driver;
+        this.locator = locator;
+        wait = new Wait(driver, locator);
+    }
+
+    private List<WebElement> getPresentElements() {
```

Comment:
private methods, as well as variables, must be under public and protected, except for static or final

public
protected
private